docs/conf.py

```diff
@@ -53,7 +53,7 @@
 
 # General information about the project.
 project = u'Treeherder'
-copyright = u'2016, Mozilla and other contributors'
+copyright = u'2017, Mozilla and other contributors'
```

Comment:
Could you just remove the year altogether? I don't think it's useful information.